npm-shrinkwrap.json

```diff
@@ -26,6 +26,20 @@
       "version": "1.2.1",
       "resolved": "https://registry.npmjs.org/extend/-/extend-1.2.1.tgz"
     },
+    "jayschema": {
```

Comment:
Yeah, looks like we should switch to `is-my-json-valid`, it has 100K downloads in the last day on npm. I guess I'll bundle that with my next pull request since it's for the schemas.